README.md

```diff
@@ -145,26 +144,32 @@ Our exception tracking tool of choice is [Rollbar](https://rollbar.com). The
 [rollbax](https://github.com/elixir-addicts/rollbax) hex package is included and
 configured to capture errors **only in the production environment**. To complete
 the Rollbar configuration, you will need to create a project in rollbar, and add
-the access token and application environment to the production environment
-variables. On heroku, you can do this with the following command:
+the access token to the production environment variables. On heroku, you can do
+this with the following command:
 
 ```bash
-$ heroku config:set APPLICATION_ENVIRONMENT=production ROLLBAR_ACCESS_TOKEN=#{access token copied from rollbar}
+$ heroku config:set ROLLBAR_ACCESS_TOKEN=#{access token copied from rollbar} DEPLOY_TAG=#{optional deploy tag to identify specific production environment}
 ```
 
-Once the environment variables are in place and the rollbar project is setup,
+Once the environment variables are in place and the rollbar project is set up,
 rollbar notifications can be created with:
```

Comment:
It just feels like if we are going to show adding in the optional env var that the first example should use the env var to keep the flow easier to follow. That or where the env var is mentioned, we add in a second command that adds the optional environment variable. That way if you didn't use it, you would use the first example command for each case and if you did use it then you would use the second example command from each case. It just feels like if the flow is consistent it will be easier to grok when others have to use it.